app/src/main/java/org/gnucash/android/util/BackupManager.java

```diff
@@ -187,7 +187,7 @@ public static void schedulePeriodicBackups(Context context) {
         Intent intent = new Intent(context, PeriodicJobReceiver.class);
         intent.setAction(PeriodicJobReceiver.ACTION_BACKUP);
         PendingIntent alarmIntent = PendingIntent.getBroadcast(context, 0, intent,
-                PendingIntent.FLAG_UPDATE_CURRENT);
+                PendingIntent.FLAG_UPDATE_CURRENT | PendingIntent.FLAG_MUTABLE);
```

Comment:
A few of those can probably be replaced by FLAG_IMMUTABLE, but I haven't tried it yet. Replacing them all causes a crash.